synapse/python_dependencies.py

```diff
@@ -104,13 +104,16 @@
     # hiredis is not a *strict* dependency, but it makes things much faster.
     # (if it is not installed, we fall back to slow code.)
     "redis": ["txredisapi>=1.4.7", "hiredis"],
+    # We pin black so that our tests don't start failing on new releases.
+    "lint": ["isort<=5.0.3", "black==19.10b0", "flake8-comprehensions", "flake8"],
```

Comment:
I think we can specify an exact version of isort now?

```suggestion
    "lint": ["isort==5.0.3", "black==19.10b0", "flake8-comprehensions", "flake8"],
```